### DISCUSSION TITLE
Provider dnsomatic updates successfully but is unhealthy

### DISCUSSION THREAD

**Discussion by witterally** [qdm12/ddns-updater#551] [Q&A] [answered]
Hello!

I'm running the latest image via Docker and while everything is seemingly updating correctly, my container is failing the health check. I'm trying to understand how/why. Below is the relevant Docker inspect:

            "Health": {
                "Status": "unhealthy",
                "FailingStreak": 3,
                "Log": [
                    {
                        "Start": "2023-10-25T13:20:30.349060042-05:00",
                        "End": "2023-10-25T13:20:30.577780758-05:00",
                        "ExitCode": 1,
                        "Output": "2023-10-25T18:20:30Z ERROR program is unhealthy: lookup all.dnsomatic.com on 192.168.125.1:53: no such host\n\n2023-10-25T18:20:30Z INFO Shutdown successful\n"
                    },
                ]
DNS-O-Matic uses all.dnsomatic.com as a catch all to update all services at once. I don't believe you can actually ping it. I could be misunderstanding it though.

Thank you!

**Comment by witterally** [comment, marked as answer]
I saw the update [here](https://github.com/qdm12/ddns-updater/issues/495) and believe my issue is fixed now as well. My container has remained healthy and the IP is no longer forcing an update when it remains the same as previously reported. This is confirmed when I go to dnsomatic.com and see the "last update" date/time is from when I started the container and not each five minute increment.

**Reply by qdm12** [reply]
Awesome :+1:
By the way, are all dnsomatic domain names dnsomatic.com? Or not always 🤔? Thanks!

**Reply by witterally** [reply]
I've never had a use case not to use dnsomatic.com but it sounds like it is possible in a blurb from their documentation? Just to note, their docs don't split out host and domain, just the full hostname. 

**hostname**

Hostname you wish to update. To update **all** services registered with DNS-O-Matic to the new IP address, hostname may be omitted or set to all.dnsomatic.com (useful if required by client). _This field is also used for services that use different names for the unique identifier of the target being updated (ex. freedns.afraid.org, TZO)._ DNS-O-Matic will format the update string appropriately for each supported service at distribution.
